### ISSUE TITLE
offline false alarm?

### ISSUE THREAD

**Issue by lidaof** [cwise89/react-detect-offline#112]
Hi,
thanks for this nice package.
I am wondering why I get the offline notice while I can still open https://ipv4.icanhazip.com/ (which shows my IP address)?
using the latest version 2.4.5

Thank you!

**Comment by nahid18** [comment]
Same issue here

**Comment by kevindekemele** [comment]
Mine keeps switching back and forth between offline and online components.

**Comment by dsacramone** [comment]
I am curious about this - is it possble to have a "time" of offline before we are determined to be "offline"? I can see users being spammed the "offline" treatment if someone is going thru a spotty area - and the offline/online switching back and forth could get annoying.. any solutions?

**Comment by neerajPortalwiz** [comment]
Same I'm having this issue, please resolve it as soon as possible..

**Comment by mdodge-ecgrow** [comment]
Yes, I am having the same issue!  I kept my console open with the network tab showing to see I could see something.  So at the times that it flashed "offline" were the red/pink lines in this screenshot.
![image](https://user-images.githubusercontent.com/57459547/215389551-8f4a131d-9e10-4022-a8ae-46ad2ff91511.png)

@cwise89 Are you still working on this library?

**Comment by mahfuzswaron** [comment]
@cwise89 when you're back with the fix, then plz notify me via this comment. for now, I am uninstalling this package from my application.

**Comment by aymengraoui** [comment]
Same issue here
